rotary/envs/qc.yaml

```diff
@@ -3,5 +3,5 @@ channels:
   - bioconda
   - defaults
 dependencies:
-#  - toulligqc=2.2.1
+  #  - toulligqc=2.2.1
```

Comment:
Best to delete this comment?

Reply:
@jmtsuji Addressed in https://github.com/rotary-genomics/rotary/pull/102